### ISSUE TITLE
[Bug][a11y]: Selectable table rows should have aria-selected attribute

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3397]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Table

### Expected behavior

Selectable table rows within a `role="grid"` should have `aria-selected` attribute that is `"true"` when the row is selected and `"false"` when the row is not selected.

### Actual behavior

Selectable table rows do not have `aria-selected` attribute to express the current state.

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error
